lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -806,14 +806,59 @@ states:
       end:
         targetState: RETURN_TO_RP
 
-  # Live in uk journey
+  # International Address journey
+  IDENTITY_START_PAGE_IA:
```

Comment:
Prefer to avoid using abbreviations in naming states (makes it not immediately obvious to follow eg. in the journey map visualiser), even if it makes the names a bit long `IDENTITY_START_PAGE_INTERNATIONAL_ADDRESS`?